_includes/corridors/us-bank.md

```diff
@@ -0,0 +1,25 @@
+{% include corridors/recipient_name.md %}
+
+## USD::Bank
+
+For United States bank payments please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  {{ recipient_name }},
+  "bank_account": "1234567890",
+  "bank_account_type": "20", // 10 for Savings, 20 for Checking
```

Comment:
I'm a bit confused, is `Checking` or `Current` (https://github.com/bitpesa/exchange4free/pull/66/files#r1126121902)? cc @thirunjuguna

Reply:
The internal implementation uses Current but for US context the documentation references Checking.

Reply:
Shouldn't we just use we out internal systems are using @evanson  ?